launch/bringup.launch.xml

```diff
@@ -4,6 +4,7 @@
   <arg name="start_gazebo" default="false" />
   <arg name="viz" default="false" />
   <arg name="stereo_pipeline_single_process" default="true" />
+  <arg name="avoid_point_cloud_padding" default="True" />
```

Comment:
Maybe, the default should still be `False`.

Reply:
I think defaulting to true is fine; this is just a playground repository.

nitpick: lowercase "true" to match the other arguments.